serverless.yml

```diff
@@ -23,6 +23,10 @@ provider:
       - ${env:AWS_SUBNET_ID_1}
       - ${env:AWS_SUBNET_ID_2}
       - ${env:AWS_SUBNET_ID_3}
+  apiGateway:
+    apiKeys:
+      - name: updateS3Key
```

Comment:
Add the stage on the apikey name.
This will make serverless create a different apikey for each stage (like described [here](https://www.serverless.com/framework/docs/providers/aws/events/apigateway#setting-api-keys-for-your-rest-api)).

And should the apikey be an `updateS3Key` or should we name it like an admin key?

Reply:
On a recent design change, this functionality will be used directly via lambda call, bypassing the `apiGateway`. So the keys were removed.